pallets/pallet-bonded-coins/src/lib.rs

```diff
@@ -256,7 +261,7 @@ pub mod pallet {
 	#[pallet::call]
 	impl<T: Config> Pallet<T>
 	where
-		<CurveParameterTypeOf<T> as Fixed>::Bits: Copy + ToFixed + AddAssign + BitOrAssign + ShlAssign,
+		<CurveParameterTypeOf<T> as Fixed>::Bits: Copy + ToFixed + AddAssign + BitOrAssign + ShlAssign + TryFrom<u128>,
```

Comment:
Where is the `TryFrom<u128>` trait used in the new code? I could not see it from the diff.

Reply:
here: https://github.com/KILTprotocol/kilt-node/blob/e94104e29ba16db3f7626488de705a92c00068b2/pallets/pallet-bonded-coins/src/curves/mod.rs#L140

On this line, the u128 is first converted to the type underlying the fixed (e.g., an i128 in case of an I75U53) and then used for division.